README.rst

```diff
@@ -181,7 +181,7 @@ of a ``30 * * * *`` (execute every 30 minutes) crontab entry you specify:
         ...     day_of_week='*',
         ...     day_of_month='*',
         ...     month_of_year='*',
-        ...     timezone=pytz.timezone('Canada/Pacific')
+        ...     timezone=zoneinfo.ZoneInfo('Canada/Pacific')
```

Comment:
will this work in older django versions?

Reply:
I was hoping the CI would run all the tests under tox...         

I'm getting myself familiar with the project and I feel there are more changes needed. ZoneInfo should work but I will test against 2.2.x, 3.2.x and 4.x just to be sure

Reply:
I've been trying to run tox and it fails in v2.2.1 as well (timeout)               
In master it's far worse with errors (since it installs [django-timezone-field 5.0](https://pypi.org/project/django-timezone-field/)

In this branch (after few more changes) Django 2.2 fails with the same timeout... so does Django 4.0....   
but Django 3.2 has a conflict installing Django

> The conflict is caused by:
>     The user requested django<4.0 and >=3.2.12
>     The user requested django<4.1 and >=4.0.2

not sure which dependency is causing this